WooCommerce/UITestsFoundation/Screens/Payments/PaymentsScreen.swift

```diff
@@ -41,6 +61,34 @@ public final class PaymentsScreen: ScreenObject {
         return self
     }
 
+    public func tapCollectPayment() throws -> Self {
+        collectPaymentButton.tap()
+        return self
+    }
+
+    public func enterPaymentAmount() throws -> Self {
+        app.enterText(text: "5")
+        nextButton.tap()
+        return self
+    }
+
+    public func takeCashPayment() throws -> Self {
+        takePaymentButton.waitForIsHittable(timeout: 10)
+        takePaymentButton.tap()
+
+        cashPaymentButton.waitForIsHittable(timeout: 10)
+        cashPaymentButton.tap()
+
+        markAsPaidButton.tap()
+        return self
+    }
+
+    @discardableResult
+    public func verifyOrderCompletedToastDisplayed() throws -> Self {
+        XCTAssertTrue(app.staticTexts.matching(NSPredicate(format: "label CONTAINS 'Order completed'")).firstMatch.exists)
```

Comment:
using `XCTAssertTrue` instead of `XCTAssertEqual` to only match text value as the complete message is "🎉 Order completed" (with the emoji) so not sure if we should include that (decided to not include it in this PR, but if anyone feels strongly that it should be included, i can update). 

also wasn't able to find an easy way to add `accessibilityIdentifier` for the toast message so using the static text value instead